Lib/email/_policybase.py

```diff
@@ -14,6 +15,12 @@
     'compat32',
     ]
 
+valid_header_name_re = re.compile("[\041-\071\073-\176]+$")
```

Comment:
Can't we have a comment saying what those octal values represent? (e.g., "equivalent to pattern "[!-9;-~]+$"). By the way `072` is `:`, so it's still printable by the way (maybe the title of the PR should be changed)

Reply:
Well, technically "Validate header name according to RFC 5322" is a very precise specification, you just have to refer to the RFC to find out what it is :).  Consequently the PR title *could* be shortened to omit the part after the rfc reference.

The reason to use the numeric version of the regex is because that's how the RFC specifies it.  I have no objection to adding a comment with the ascii version of the regex.

Reply:
> because that's how the RFC specifies

Oh I see (I don't mind keeping the octal values, just maybe a one-line comment would be nice for future readers)